legal-api/tests/unit/services/test_authorization.py

```diff
@@ -461,6 +471,16 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
         ('staff_active', Business.State.ACTIVE, 'changeOfDirectors', None,
          ['LLC'], 'staff', [STAFF_ROLE], False),
 
+        ('staff_active_allowed', Business.State.ACTIVE, 'consentContinuationOut', None,
+         ['BC', 'BEN', 'ULC', 'CC'], 'staff', [STAFF_ROLE], True),
+        ('staff_active', Business.State.ACTIVE, 'consentContinuationOut', None,
+         ['CP', 'LLC'], 'staff', [STAFF_ROLE], False),
+
+        ('staff_active_allowed', Business.State.ACTIVE, 'continuationOut', None,
+         ['BC', 'BEN', 'ULC', 'CC'], 'staff', [STAFF_ROLE], False),
+        ('staff_active', Business.State.ACTIVE, 'continuationOut', None,
+         ['CP', 'LLC'], 'staff', [STAFF_ROLE], False),
+
```

Comment:
please add this test for general user to confirm they can't file a `continuationOut`